src/app/AppHeader.vue

```diff
@@ -95,6 +104,9 @@ const mode = computed(() => store.getters['config/getMulticlusterStatus'] ? 'Mul
 .logo-image {
   max-height: 36px;
 }
+.gh-star {
+  height: 20px;
+}
```

Comment:
The reason the element takes up an unexpected amount of height is because the `.widget` in the component’s shadow DOM has `display: inline-block` but no `vertical-align: middle;`.

Unfortunately, `.widget` doesn’t have a `part` attribute so we can’t style the element using `::part` (this would work):

```css
.gh-star ::part(widget) {
  vertical-align: middle;
}
```

At this point it’s not worth spending more time on optimizing this.

(I will say however that it would probably be less maintenance overhead for us to implement this link ourselves as this package and its dependencies aren’t official so there isn’t really a guarantee of maintenance not being a problem.)

Reply:
Ok let's merge this for now at least. There is more GH things to get added potentially, so if we end up rolling our own for other ones we can pull the deps out here and roll our own/use a link for this one also.